policy/README.md

```diff
@@ -225,6 +227,7 @@ An individual `Rule` object is defined by the following fields:
 | `propulsion_types` | `propulsion_type[]`         | Optional   | Applicable vehicle propulsion types, default "all". |
 | `minimum`          | integer                     | Optional   | Minimum value, if applicable (default 0) |
 | `maximum`          | integer                     | Optional   | Maximum value, if applicable (default unlimited) |
+| `rate_amount`             | integer                     | Optional   | The amount of a rate applied when this rule applies, if applicable (default zero). A positive integer rate amount represents a fee, while a negative integer represents a subsidy. Rate amounts are given in the `currency` defined in the [Policy](#policy). |
```

Comment:
Should this be a float? ISO 4217 Alphabetic Currency Code for the US would be dollars but fees are often cents.

Reply:
The linked to page (from Provider) says "All costs should be given as integers in the currency's smallest unit"